torch/_inductor/autotune_process.py

```diff
@@ -116,8 +116,7 @@ class TensorMeta:
 
     @classmethod
     def from_irnodes(
-        cls,
-        irnodes: Union[Tuple[TensorMeta, ...], List[TensorMeta], ir.Layout],
+        cls, irnodes: torch.Tensor,
```

Comment:
This is incorrect, irnodes is not always Tensors, it can be tuple/list nested arbitrarily, can be ir.Layout, can be Tensor

Reply:
Hi @gmagogsfm! Thanks for pointing out. Have updated the PR.